eo-maven-plugin/src/main/resources/org/eolang/maven/pre/rename-tests-inners.xsl

```diff
@@ -39,6 +39,8 @@ SOFTWARE.
           <xsl:text>ω</xsl:text>
           <xsl:value-of select="$class/@ancestors"/>
           <xsl:value-of select="substring-after($class/@name, concat($class/@parent, '$'))"/>
+          <xsl:text>-hash-</xsl:text>
```

Comment:
@levBagryansky why do we need this `-hash-`?

Reply:
@maxonfjvipon it describes what next letters mean. We can remove it, but `hash` looks better as for me

Reply:
@levBagryansky ok, let's leave it for now